### ISSUE TITLE
Plug in is not doing anything

### ISSUE THREAD

**Issue by JenFau** [windingwind/zotero-better-notes#122]
Hi, I just downloaded and installed this today (0.6.27), and although it's added new options to the menus such as 'open new main note', 'create new main note', nothing happens when I click them and I can't get anything else in the user guide to work. I can't see how to open a workspace, either. Could you help, please?

**Comment by a5116638** [comment]
I had the same problem

**Comment by windingwind** [comment]
Could you please share a debug output via Help->Debug Output->View Output?
And, does this happen to only the latest version? Does reinstalling the plugin solve this problem?

**Comment by RainEggplant** [comment]
> Could you please share a debug output via Help->Debug Output->View Output? And, does this happen to only the latest version? Does reinstalling the plugin solve this problem?

I also encountered this problem after the plugin upgraded to the latest (0.6.27) version. The "Open workspace" button disappears and I cannot open or create any main note. Reinstalling does not solve this problem.

**Comment by windingwind** [comment]
@RainEggplant 
Please share a debug output via Help->Debug Output->View Output

**Comment by RainEggplant** [comment]
```
[JavaScript Error: "XML Parsing Error: no root element found
Location: https://<webdav_username>:****@<webdav_host>/zotero/
Line Number 1, Column 1:" {file: "https://<webdav_username>:****@<webdav_host>/zotero/" line: 1}]

[JavaScript Error: "TypeError: Zotero.Knowledge4Zotero is undefined" {file: "chrome://zotero/content/standalone/standalone.xul" line: 1}]
oncommand@chrome://zotero/content/standalone/standalone.xul:1:1

version => 6.0.13, platform => Win32, oscpu => Windows NT 10.0; WOW64, locale => zh-CN, appName => Zotero, appVersion => 6.0.13, extensions => ZotFile (5.1.1, extension), Zotero Citation Counts Manager (1.3.0, extension), Zotero Save with Tag (0.1.1, extension), DOI Manager (1.4.2, extension), Zotero Tag (0.2.13, extension), Zotero PDF Translate (0.8.22, extension), Zotero LibreOffice Integration (6.0.0.SA.6.0.13, extension), Zotero Word for Windows Integration (6.0.0.SA.6.0.13, extension), Zotero Better Notes (0.6.27, extension)
```

Zotero upgraded itself yesterday. This may also be the problem.

**Comment by windingwind** [comment]
@RainEggplant Does downgrade to v0.6.26 solves the bug?

**Comment by RainEggplant** [comment]
> @RainEggplant Does downgrade to v0.6.26 solves the bug?

Yes, this solves the problem.

**Comment by windingwind** [comment]
I replaced the v0.6.27 xpi resource. Somehow part of the scripts are missing after packing.

**Comment by windingwind** [comment]
Please download again: https://github.com/windingwind/zotero-better-notes/releases/tag/0.6.27

**Comment by RainEggplant** [comment]
Working fine now🎉! Thank you!

**Comment by windingwind** [comment]
This is a bug of build.js, I've fixed it.
To make sure users updated to v0.6.27 are not affected, I released a v0.6.28, though with no functional updates.

**Comment by JenFau** [comment]
> Please download again: https://github.com/windingwind/zotero-better-notes/releases/tag/0.6.27
> 
> @RainEggplant @JenFau @a5116638

Thank you!! Working now - and thank you for such a great plug in
